src/test-resources/basic-axe-v3.2.2-sarif-v2.1.2.sarif

```diff
@@ -0,0 +1,768 @@
+{
+    "version": "2.1.2",
```

Comment:
I don't think this is quite right; the "version" property is supposed to be 2.1.0, and there should be a "$schema" property reflecting the true 2.1.2 version.

You can test against the sarif schema using the sarif multitool by:
* Installing the latest .NET Core SDK from https://dotnet.microsoft.com/download
* From your command prompt, running:
  ```
  dotnet tool install --global Sarif.Multitool
  sarif validate ./basic-axe-v3.2.2-sarif-v2.1.2.sarif
  ```

Reply:
(For now, I think we can ignore the SARIF1008 errors and defer dealing with them for a later PR)

Reply:
I fixed all the format errors that I saw, but I didn't get any SARIF1008 errors when I ran this. Is that fine, or does that mean I'm running a different version of the Sarif Multitool?